build.gradle

```diff
@@ -246,7 +246,7 @@ def getGitHash = { ->
 
 
 docker {
-    name "gcr.io/broad-jade-dev/${bootJar.baseName}:latest${System.env.GOOGLE_CLOUD_PROJECT}"
+    name "gcr.io/broad-jade-dev/${bootJar.baseName}:${System.env.TAG}"
```

Comment:
TAG is pretty generic for an environment variable. Maybe DATAREPO_TAG or similar?

Reply:
Yep I was thinking that this morning -- will change.